File: backend/services/MarketplaceService/src/MarketplaceService.Domain/Events/Sortie/SortieRecordedEvent.cs
using EventsContracts.EventsContracts;

namespace MarketplaceService.Domain.Events.Sortie
{
    public class SortieRecordedEvent : ISortieRecordedEvent
    {
        public List<ISortieItem> SortieItems { get; set; } = new List<ISortieItem>();
        public DateTime Date { get; set; }
    }
}

File: backend/services/stockservice/src/main/java/com/api/stockservice/domain/event/Sortie/SortieRecordedEvent.java
package com.api.stockservice.domain.event.Sortie;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.OffsetDateTime;
import java.util.List;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class SortieRecordedEvent {
    List<SortieItem> sortieItems ;
    private OffsetDateTime date;
}
